src/tabset/tabset.spec.ts

```diff
@@ -225,6 +229,55 @@ describe('ngb-tabset', () => {
          expectTabs(fixture.nativeElement, [true], [true]);
        });
      })));
+
+    it('should change active tab by using a button click', async(inject([TestComponentBuilder], (tcb) => {
```

Comment:
I would rather say "should change active tab by calling select on an exported directive instance" since using buttons is just one way of invoking the `select` method.